signs-work-order-map/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "signs-work-order-map",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@mapbox/mapbox-gl-geocoder": "^5.0.3",
+    "@types/mapbox__mapbox-gl-geocoder": "^5.0.0",
+    "@types/mapbox-gl": "^3.4.1",
+    "mapbox-gl": "^3.10.0",
+    "next": "15.2.4",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
+    "react-map-gl": "^8.0.1",
+    "react-map-gl-geocoder": "^2.2.0"
```

Comment:
this package's dependencies seems to be responsible for a handful of vulnerability warnings. i see we are instead using `@mapbox/mapbox-gl-geocoder` in the VZE. that's because of the difference in `react-map-gl` versions?

Reply:
changes they've made over the last few versions make this confusing, but I think that this dependency can be removed. their latest geocoder example doesn't show it https://visgl.github.io/react-map-gl/examples/mapbox/geocoder. also just cool to see that they're on to version 8!

Reply:
yeah! thank you Chia for sorting out our upgrade path to v8 in the VZE 😇